ts/features/lollipop/saga/index.ts

```diff
@@ -72,6 +73,14 @@ export function* checkLollipopSessionAssertionAndInvalidateIfNeeded(
   maybePublicKey: O.Option<PublicKey>,
   maybeSessionInformation: O.Option<PublicSession>
 ) {
+  // The actual function is called after a successful login.
```

Comment:
I do not think that (in this comment) we are explicitly explaining why we do not need to check the lollipop assertion and why we have placed this code here. My suggestion would be something like:

"When using the test idp to login, no authentication data nor lollipop key are saved / used / sent. Therefore, we must not check for the lollipop assertion (when this function is called after a successful test idp login) otherwise the (test) user is immediately logged-out"

Reply:
Much better. I've also track a future refactoring with a Jira issue. Thank you!